postbuild_fixups.sh

```diff
@@ -5,8 +5,8 @@ version=${1?Please specify version}
 # src/DashTextareaAutocomplete
 declare -a moduleregs=(\
 's/https:\/\/unpkg.com\/dash_textarea_autocomplete/https:\/\/unpkg.com\/dash-textarea-autocomplete/' \
-# 's/realpath(joinpath( @__DIR__, \"..\", \"deps\"))/artifact\"dash_textarea_autocomplete_resources\"/' \
-# 's/using Dash/using Dash, Pkg.Artifacts/' \
+'s/realpath(joinpath( @__DIR__, \"..\", \"deps\"))/joinpath(artifact\"dash_textarea_autocomplete_resources\",\"deps\")/' \
```

Comment:
So if I understand correctly. Alternatively, we could have

```diff
--- a/package.json
+++ b/package.json
@@ -15,7 +15,7 @@
     "build:js": "webpack --mode production",
     "build:backends": "dash-generate-components ./src/lib/components dash_textarea_autocomplete -p package-info.json --r-prefix '' --jl-prefix ''",
     "build:backends-activated": "(. venv/bin/activate || venv\\scripts\\activate && npm run build:py_and_r)",
-    "build:create-tar-file-from-deps": "tar -czf ./deps.tar.gz ./deps && mv ./deps.tar.gz ./deps",
+    "build:create-tar-file-from-deps": "tar -czf ./deps.tar.gz ./deps/* && mv ./deps.tar.gz ./deps",
```

and cut the extra `"deps"` piece here?

cut that `"deps"` piece and

Reply:
Yes! We could also skip the `joinpath` step in `src/DashTextareaAutocomplete` 

I used it to test the `v1.2.1`'s tarball, but we can do it differently.

On the other hand we know the `/deps/` approach works. We'd have to make sure the path will still work without it for the release